### PR TITLE
Lord-x7 card

### PR DESCRIPTION
**Pull request by Lord-x7**
Added my card to index.html file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,36 @@ <h2>A project for first-time contributions</h2>
...
+          <p class="about">I'm a student from India .Currently focusing On Data Structures and Algorithms,Linux and Python develeopment</p>
+          <div class="resources">
+            <p>3 Useful Resources: 
+            </p>
+            <ul>
+              <li>
+                <a href="www.linkedin.com" target="_blank" title="First Resource">Linkedin</a>
+              </li>
+              <li>
+                <a href="https://www.google.com/search?q=florian+dedov+python+bible&rlz=1C1CHBF_enIN1003IN1003&oq=florian+dedov+python+bible&aqs=chrome..69i57.10361j0j1&sourceid=chrome&ie=UTF-8&si=AC1wQDAtx2BbmyBmykHcHVhKaK47clF_XdL8sYdO8Trpk4O-6hko1BlMx51g9LBviEWcmLSfS3xRLaHkRtdX167nKuNbwXZS4v8z6AfcaQMIL_cdeyewnDkmUd5WSHlN2WcpfDqErcT7qVU5da3RDQWZBlfQ-h2GuLxOW-fV-Na_vlC7ryJXmfizHNbvCLQncNrCFNi0a_vFgKV94qYlRlboCVwNLRCl3HD2l3WMAmKSbsMiEtxJI8Qvt1GkDE6oE9OzgVWv8veLZl0kO0epJbnsuP7ehc2ZLgGhdI0ovtD5NzsxMW92JIA%3D&ictx=1&ved=2ahUKEwij_Krrlqb7AhWCSWwGHbZaBQIQnZMFegQIZBAC" target="_blank" title="Second Resource">Book to learn Python</a>
```
Comment (TenTraicion) [inline index.html:138]: Remove the google search and put an actual link that may lead to the book itself, ex: author website/pdf download link etc.

File: index.html [hunk 2]
```diff
@@ -115,6 +115,36 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" www.geeksforgeeks.com="_blank" title="Third resource">Resource for Data Structures and Algorithms</a>
```
Comment (TenTraicion) [inline index.html:141]:
```suggestion
                <a href="www.geeksforgeeks.com" target="_blank" title="Third resource">Resource for Data Structures and Algorithms</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Lord-x7, Thanks for the contribution. kindly make the required changes and commit.

File: index.html (hunk 1 above)
Comment (Lord-x7) [reply index.html]: Done

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@Lord-x7, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**